apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -121,6 +121,7 @@ export KUMA_ATTACHMENTS_AWS_S3_CUSTOM_DOMAIN ?= ""
 export KUMA_ATTACHMENTS_USE_S3 ?= False
 export KUMA_CELERY_TASK_ALWAYS_EAGER ?= False
 export KUMA_CELERY_WORKER_MAX_TASKS_PER_CHILD ?= 0
+export KUMA_CONN_MAX_AGE ?= 0
```

Comment:
This is a huge change. What's the reasoning for it? 
It seems like a dangerous setting.

Reply:
@peterbe Just to add to what @limed said, we only switched to a `CONN_MAX_AGE` greater than zero (60s in our case) about 2 years ago after we finally updated to a Django version that supported it, and it didn't make much difference (see https://bugzilla.mozilla.org/show_bug.cgi?id=1434296). In the current situation, we're seeing about an average of 5ms extra response time when setting it back to zero, and we thought it was an acceptable price to pay in order to avoid DB connections that are never closed on the Django side when using `gevent` (which surprisingly, we never saw with `meinheld`).

Reply:
@limed @peterbe After a day of running with `gevent`, the average cost of `CONN_MAX_AGE=0` seems to be something more like 10ms at peak load of 10K rpm.